pkg/controller/subnet.go

```diff
@@ -596,25 +596,24 @@ func (c *Controller) checkSubnetConflict(subnet *kubeovnv1.Subnet) error {
 }
 
 func (c *Controller) updateSubnetDHCPOption(subnet *kubeovnv1.Subnet, needRouter bool) error {
-	var dhcpOptionsUUIDs *ovs.DHCPOptionsUUIDs
-	var err error
-	var mtu int
-	if subnet.Spec.Vlan != "" {
-		mtu = 1500
-	} else {
+	mtu := util.DefaultMTU
+	if subnet.Spec.Vlan == "" {
 		switch c.config.NetworkType {
-		case util.NetworkTypeVxlan:
-			mtu = 1500 - util.VxlanHeaderLength
+		case util.NetworkTypeVlan:
+			// default to geneve
```

Comment:
I think in vlan case, the mtu should be 1500, which should not be the same with geneve case.